src/content/en/2020/markup.md

```diff
@@ -626,34 +704,42 @@ Using `target="_blank"` is known as a [security vulnerability](https://mathiasby
 
-When we don’t need to [cover 14 years](../2019/markup) for analysis but only 1, one could almost get the impression that HTML is rather inert, that not much changes.
+## Conclusion
 
-Yet what we observe with this year’s data is that there’s lots of movement at the bottom and near the shore of said sea of HTML: We approach near-complete adoption of living HTML, are quick to prune our pages of fads (like Google and Facebook widgets), and we’re fast in adopting and shunning frameworks (both Angular and AMP—though a “component framework”—seem to have significantly lost in popularity, likely for solutions like React and Vue).
+When we don't need to [cover 14 years](../2019/markup) for analysis but only 1, one could almost get the impression that HTML is rather inert, that not much changes.
 
-And still, there are no signs we exhausted the options HTML gives us. The median of 30 different elements used on a given page, which is roughly a quarter of the elements HTML provides us with, suggests a rather one-sided use of HTML. That is supported by the immense popularity of elements like `div` and `span`, and no custom elements to potentially meet the demands that these two elements may represent. Unfortunately we couldn’t validate each document in the sample (yet we learned—to be used with caution—that of W3C-tested documents, [79% have errors](https://github.com/HTTPArchive/almanac.httparchive.org/issues/899#issuecomment-717856201)), but after everything we’ve seen it looks like we’re still far from mastering the craft of HTML.
+Yet what we observe with this year's data is that there's a lot of movement at the bottom and near the shore of said sea of HTML. We approach near-complete adoption of living HTML, are quick to prune our pages of fads like Google and Facebook widgets, and we're fast in adopting and shunning frameworks. Both Angular and AMP, though "component frameworks", seem to have significantly lost in popularity, likely for solutions like React and Vue.
 
-That suggests us to close with an appeal:
+And still, there are no signs we exhausted the options HTML gives us. The median of 30 different elements used on a given page, which is roughly a quarter of the elements HTML provides us with, suggests a rather one-sided use of HTML. That is supported by the immense popularity of elements like `div` and `span`, and no custom elements to potentially meet the demands that these two elements may represent. Unfortunately, we couldn't validate each document in the sample. However, anecdotally and to be taken with caution, we learned that [79%](https://github.com/HTTPArchive/almanac.httparchive.org/issues/899#issuecomment-717856201) of W3C-tested documents have validation errors. After everything we've seen, it looks like we're still far from mastering the craft of HTML.
```

Comment:
1. I don’t like the break and pause for validation (“…sample. However”)—it drags the whole point out for little value. Could you join these to form one unit?

2. Please replace “we’ve seen” by “we’re seeing” (consistency in tense).

Reply:
1. Could you suggest the change you're looking to see?

2. Not consistent with "we learned" in the previous sentence. Should we also replace that or keep them both?

Reply:
1. I think a semicolon would help: “…sample; however…”.

2. Probably my suggested change isn’t that important; now I rather wonder we should instead go for “we’ve learned” 😄 You decide.